### ISSUE TITLE
[Bug] Can drag folder to gitbutler window but can't add project

### ISSUE THREAD

**Issue by LaoshuBaby** [gitbutlerapp/gitbutler#4084]
I can drag a folder to this window, but nothing happened, and I only have to add it manually.

https://github.com/gitbutlerapp/gitbutler/assets/42690037/0b3c7d7a-6f3e-4cc8-9588-27fabda5f950

OS is Windows 10 Build 19045, and haven't tested on other machine.

**Comment by Byron** [comment]
This seems like a very intuitive thing to do, thanks for sharing!

CC @PavelLaptev in case you feel the same *and* think it's easy to implement.

**Comment by PavelLaptev** [comment]
@Byron yes. It would be nice. Can also add a state when drag a folder and hover it over the app window with the copy something like "Drag and drop your repo folder to add a new project." Just tag me in your PR when you finish, I'll make a design for it.

**Comment by PavelLaptev** [comment]
Closing the issue.
The feature should be in the next release.
